items.json

```diff
@@ -10531,9 +10531,9 @@
     "name": "Fine Steel Menavlion",
     "culture": "Empire",
     "type": "Polearm",
-    "price": 34349,
-    "weight": 1.1,
-    "tier": 15.0723047,
+    "price": 18041,
+    "weight": 1.29,
+    "tier": 10.62182,

```

Comment:
tier too high

Reply:
thought you would let it slide with these stats its really not overpowered at all

![image](https://user-images.githubusercontent.com/109090029/202330967-599b8c75-a2a8-48e5-a2e8-f80925a0bdd6.png)

Reply:
Ill change it right now to be under 10